ghost/members-api/lib/services/member-bread.js

```diff
@@ -254,6 +254,10 @@ module.exports = class MemberBREADService {
         let model;
 
         try {
+            const attribution = await this.memberAttributionService.getAttributionFromContext(options?.context);
+            if (attribution) {
+                data.attribution = attribution;
```

Comment:
This should be of type `Attribution` rather than `AttributionResource`

Reply:
🤔 What we pass down to the member create is actually just a plain resource object and not `Attribution` object, which is why I had added `AttributionResource` here. We don't pass down the `Attribution` object to the event, if we use `.build(...)` here would that need a `fetchResource` again down the line ?

Reply:
Hmm, the definition for the `create` and `linkSubscription` methods both expect an Attribution object - so I think we either need to fix it here, or in there!

Reply:
Ah yea so it will need an update in `create` and `linkSubscription` i think, will update.

Reply:
Updated.

Reply:
Thank uu!!